contributing/development.md

```diff
@@ -69,7 +69,7 @@ For more info about working with this site, check out these READMEs:
 - [data/README.md](../data/README.md)
 - [data/reusables/README.md](../data/reusables/README.md)
 - [data/variables/README.md](../data/variables/README.md)
-- [includes/liquid-tags/README.md](../includes/liquid-tags/README.md)
+- [tests/fixtures/README.md](tests/fixtures/README.md)
```

Comment:
```suggestion
- [src/content-render/README.md](src/content-render/README.md)
```

Reply:
This should be the correct link to the `content-render` section that discusses liquid tags.